tests/browser/harvest.browser.js

```diff
@@ -95,20 +95,22 @@ function validateUrl (t, actualUrl, expectedUrlTemplate, message) {
   // Also get the ck parameter value from the actual URL
   let queryString = actualUrl.split('?')[1]
   let pairs = queryString.split('&')
-  let submissionTimestamp
+  let submissionTimestamp, sessionId;
   let ckValue = '1'
   for (var i = 0; i < pairs.length; i++) {
     let pair = pairs[i].split('=')
     if (pair[0] === 'rst') {
       submissionTimestamp = pair[1]
     } else if (pair[0] === 'ck') {
       ckValue = pair[1]
+    } else if (pair[0] === 's') {
+      sessionId = pair[1];
     }
   }
 
   // In addition to replacing timestamp, add in the ck parameter which goes after timestamp
-  let expectedUrl = expectedUrlTemplate.replace('{TIMESTAMP}', submissionTimestamp + '&ck=' + ckValue)
-
+  let expectedUrl = expectedUrlTemplate.replace('{TIMESTAMP}', `${submissionTimestamp}&ck=${ckValue}&s=${sessionId}`)
```

Comment:
Just want to confirm, Is `s=0` going to be our way of handling **both** cases of disabling? 
ie:
1.  if the end-user disables this behavior
3. if the customer disables this behavior?

Reply:
That appears to be the case according to the doc, yes, and I don't see a need to distinguish them.
As it stands, from talking with @aubreymasten before, no.1 isn't even supported right now as in there's no mechanism to flip the switch during runtime right now while on the same page.

Reply:
Right, but `#1` is on our board to implement for the current MMF so we should consult with John if that is still needed.